models/event.js

```diff
@@ -60,7 +60,11 @@ const MODEL = {
         .example({
             nodes: [{ name: '~commit' }, { name: 'main' }, { name: 'publish' }],
             edges: [{ src: '~commit', dest: 'main' }, { src: 'main', dest: 'publish' }]
-        })
+        }),
+    pr: Joi
```

Comment:
And probably better to make  `pr` an object and have `url` as a field inside. Later on if we want to add new stuff related to pr to the event, then we can easily do that by adding stuff to this object.